jobserver/models.py

```diff
@@ -318,6 +318,7 @@ class Workspace(models.Model):
     repo = models.TextField(db_index=True)
     branch = models.TextField()
     is_archived = models.BooleanField(default=False)
+    will_notify = models.BooleanField(default=False)
```

Comment:
Given that the notification goes to the user who created the JobRequest, it seems a bit odd to me to have this as a property of the workspace. Wouldn't it make more sense to have it on either the JobRequest (so a flag that was set as part of the job request form) or as a per-user setting?

Reply:
This came from @sebbacon in #214.  For a first pass I think this tracks, clicking the "turn on notifications" every time you submit a JobRequest sounds a bit tedious and getting back to a JobRequest is not something I've promoted in the UI very much.

I can see the per-user argument but I don't have a strong opinion on either.

@sebbacon – thoughts?

Reply:
In real-life use people are very unlikely ever to do anything other than subscribe to a workspace or not, but I think we can assume we'll be refining this for various override-type situations in the future.

I'd have thought the most future-proof data model would be for a notification to be per-user-per-jobrequest (i.e. a M2M "notification" model twixt jobrequest and user.  Notifications could then have extra metadata like "notification method" or "max rate" or "frequency" etc; the fact it's typically subscribed to on a workspace level would be handled separately.

But this is all fine for now